Resources/Prototypes/SS220/Entities/Structures/barricade.yml

```
@@ -0,0 +1,423 @@
+    drawdepth: BlastDoors
+    noRot: false
+  - type: Appearance
+  - type: AtmosExposed
+  - type: Physics
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.5,-0.5,0.39,-0.36"
+        mask:
+        - FullTileMask
+        layer:
+        - WallLayer
+ #Баррикады       
+- type: entity
+  id: BarricadeWooden
+  description: Баррикада из уплотнённых деревянных досок. Защищает вас от чужих глаз. 
+  parent: EstBaseBarricade
+  name: деревянная уплотненная баррикада
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: Sprite
+    sprite: SS220/Structures/barricades.rsi
+    state: barricade_wooden
+    noRot: false 
+  - type: Construction
+    graph: BarricadeWooden
+    node: barricadefull
+  - type: Tag
+    tags:
+    - Wooden
+  - type: Flammable
+    fireSpread: true
+    damage:
+      types:
+        Heat: 2 
+  - type: FireVisuals
+    sprite: Effects/fire.rsi
+    normalState: 1
+  - type: Damageable
+    damageModifierSet: Wood
+    damageContainer: StructuralInorganic
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 90
+      behaviors:
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          MaterialWoodPlank1:
+            min: 3
+            max: 3
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+        
+- type: entity
+  id: BarricadeSteel
+  description: Баррикада из стальных листов. Защищает вас от чужих глаз. Способен выдержать пулю.
+  parent: EstBaseBarricade
+  name: стальная баррикада
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: Sprite
+    sprite: SS220/Structures/barricades.rsi
+    state: barricade_steel
+    noRot: false 
+  - type: Construction
+    graph: BarricadeSteel
+    node: barricadefull
+  - type: Tag
+    tags:
+    - Steel
+  - type: Damageable
+    damageModifierSet: StructuralMetallic
+    damageContainer: StructuralInorganic
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 180
+      behaviors:
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          SheetSteel1:
+            min: 3
+            max: 3
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+        
+- type: entity
+  id: BarricadePlasteel
+  description: Баррикада из листов пластали. Защищает вас от чужих глаз. Способен выдержать взрыв C4.
+  parent: EstBaseBarricade
+  name: укрепленная баррикада
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: Sprite
+    sprite: SS220/Structures/barricades.rsi
+    state: barricade_plasteel
+    noRot: false 
+  - type: Construction
+    graph: BarricadePlasteel
+    node: barricadefull
+  - type: Tag
+    tags:
+    - Plasteel
+  - type: Damageable
+    damageModifierSet: StructuralMetallic
+    damageContainer: StructuralInorganic
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 440
```

```suggestion
        damage: 440
```
имхо многовато хп для постройки из пары кусков стали